### PR TITLE
config: Update the default stitch

### PR DESCRIPTION
**Pull request by nlsun**

### CODE REVIEW

File: config.spec [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
-(label "purple" (makeList 1 (docker "nginx")))
-
-(connect (list 1024 65535) "red" "blue")
-(connect (list 1024 65535) "blue" "red")
-
-(place (labelRule "exclusive" "yellow") "purple")
+(define AdminACL (list "local" "127.0.0.1/32"))
+
+
+// This is a global constant, which is discouraged in favor of local
```
Comment (kklin) [inline config.spec]: Now that we have `set`, `define`s are no longer constants.

File: config.spec [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
+  (connect (list 1024 65535) "red" "blue")
+  (connect (list 1024 65535) "blue" "red")
+
+  (place (labelRule "exclusive" "yellow") "purple"))
+
+
+// This is a function that launches machines.
+(define (createMachines nMaster nWorker)
+  (machineAttribute
+    (list (machineAttribute (makeList nMaster (machine)) (role "Master"))
```
Comment (kklin) [inline config.spec]: Don't think we need a `machineAttribute` here, could just do `(machine (role "Master"))`.

File: config.spec (hunk 1 above)
Comment (nlsun) [inline config.spec]: oh, does set mutate a defined variable?

**Comment by ejj** [comment]
Merged

**Pull request closed without merging**